actions/storage_controller.go

```diff
@@ -14,6 +14,7 @@ import (
 
 var (
 	ErrVirtualDiskManagerUtilNotIdentified = errors.New("virtual disk management utility not identifed")
+	ErrDiskkManagerUtilNotIdentified       = errors.New("disk management utility not identifed")
```

Comment:
```suggestion
	ErrDiskManagerUtilNotIdentified       = errors.New("disk management utility not identifed")
```

Reply:
Removed, the 1st approach was to use a tool, but Go has everything needed to program it natively.